RUFAS/routines/animal/pen.py

```diff
@@ -529,7 +519,10 @@ def calc_daily_walking_dist(self):
         """
         Sets the daily walking distance for the cows in the pen (if any).
         """
-        if "Cow" in self.classes_in_pen:
+        if (
+            self.animal_combination == AnimalCombination.LAC_COW
+            or self.animal_combination == AnimalCombination.CLOSE_UP
+        ):
```

Comment:
With if statements like this, it makes it often makes the code cleaner if the condition is extracted into its own line and named explicitly:
```suggestion
cows_are_in_pen = self.animal_combination in {AnimalCombination.LAC_COW, AnimalCombination.CLOSE_UP}
if cows_are_in_pen:
```

Reply:
Also, this change does not appear to be explicitly related to deprecating `update_classes_in_pen`, why is this needed?